app.go

```diff
@@ -224,3 +226,75 @@ func executeQuote(cmd command) bool {
 	// send the quote to the user
 	return true
 }
+
+func executeBuy(cmd command) bool {
+	//Gotta check users money and add a reserved portion
+	account := accountStore.GetAccount(cmd.UserID)
+
+	if account == nil {
+		log.Noticef("User %s does not have an account", account)
+	}
+
+	stockSymbol := cmd.Args[0]
```

Comment:
Potentially, but if we have a non standard number of args, the code will fail before this point. That logic should probably sit in the parse cmd code.